digitalidentity/share_session_notification_builder.go

```diff
@@ -0,0 +1,62 @@
+package digitalidentity
+
+import (
+	"encoding/json"
+)
+
+// ShareSessionNotification specifies a ShareSessionNotification for ShareSession
+type ShareSessionNotification struct {
+	url       string
+	method    string
+	verifyTls bool
+	Headers   map[string][]string
```

Comment:
```suggestion
	headers   map[string][]string
```

just to be consistent with the pattern of what you're doing